### PR TITLE
Add icons for SchildiChat, BootBoi & NC-Photos

### PR DESCRIPTION
**Pull request by fischer-felix**
## Description

Adds icons for:
- https://github.com/BootBoi/android-app
- https://github.com/SchildiChat/SchildiChat-android
- https://gitlab.com/nkming2/nc-photos

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* SchildiChat (`de.spiritcroc.riotx`)
* BootBoi (`com.elektropepi.bootboi`)
* Photos (for Nextcloud) (`com.nkming.nc_photos.gitlab`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -604,6 +606,7 @@
     <icon drawable="@drawable/nextcloud_notes" package="it.niedermann.owncloud.notes" name="Nextcloud Notes" />
     <icon drawable="@drawable/nextcloud" package="com.nextcloud.android.beta" name="Nextcloud Dev" />
     <icon drawable="@drawable/nextcloud" package="com.nextcloud.client" name="Nextcloud" />
+    <icon drawable="@drawable/nextcloud_notes" package="it.niedermann.owncloud.notes" name="Nextcloud Notes" />
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:609]: This line is already added in line 604 / 606. Could you remove it?

**Review by Chefski — CHANGES_REQUESTED** [review]
Hiya! Thanks for contributing! I've got some changed to request:

`bootboi.svg` is smaller than the 160x160px content area and has a hidden square which isn't being used in the icon. Could you size up the icon and remove the hidden square (just to keep the SVG minimal)?
image
image


<br>
<br>

`schildichat.svg` has strokes of 11.3 and opacity isn't at 100% for all lines. Could you change the stroke to 12px and ensure each line has 100% opacity?
image

**Comment by fischer-felix** [comment]
Thanks for the heads-up, had forgotten to remove my reference from the bootboi icon.
Both things should be fixed now!

**Review by Chefski — APPROVED** [review]
Thanks for making the changes

**Pull request merged**